spec/integration/object_migration_spec.rb

```diff
@@ -27,6 +27,18 @@
       expect(subject.edit_users).to include("jilluser@example.com")
     end
 
+    describe "objects with Om datastreams" do
+      let(:mover) { FedoraMigrate::ObjectMover.new source, ExampleModel::OmDatastreamExample.new }
```

Comment:
This is subjective, but I'd prefer to see explicit parens for the outer `#new` call here. It confused my eyes for a minute.